dict.ko.css

```diff
@@ -10,6 +10,10 @@
  * governing permissions and limitations under the License.
  */
 
+ h1, h2, h3, h4, h5, p {
```

Comment:
Why not overlay the `font-family` at the `body` level?

Reply:
Because the request was only for headings, paragraphs and image captions (p tag too). I did not want to change it for everything. But you might be right...